homeassistant/components/smappee_official/binary_sensor.py

```diff
@@ -0,0 +1,224 @@
+"""Support for monitoring a Smappee appliance binary sensor."""
```

Comment:
Please limit the first PR to only one platform. Further platforms should come later one platform per PR.

Reply:
The first PR has been limited to only sensor entities in commit [#7a20280](https://github.com/home-assistant/core/pull/36445/commits/7a20280f840f596199da42edfc0322dba10daeb6).